app/views/foods/index.html.erb

```diff
@@ -1,14 +1,35 @@
-<p style="color: green"><%= notice %></p>
+<div class="w-full">
+  <% if notice.present? %>
+    <p class="py-2 px-3 bg-green-50 mb-5 text-green-500 font-medium rounded-lg inline-block" id="notice"><%= notice %></p>
+  <% end %>
 
-<h1>Foods</h1>
+  <div class="flex justify-between items-center">
+    <h1 class="font-bold text-4xl">Foods</h1>
+    <%= link_to 'New food', new_food_path, class: "rounded-lg py-3 px-5 bg-blue-600 text-white block font-medium" %>
+  </div>
 
-<div id="foods">
-  <% @foods.each do |food| %>
-    <%= render food %>
-    <p>
-      <%= link_to "Show this food", food %>
-    </p>
-  <% end %>
+  <div id="foods" class="min-w-full flex flex-col items-center justify-center mt-10">
+    <table class="min-w-full border">
+      <thead>
+        <tr>
+          <th class="px-6 py-3 bg-gray-200 text-gray-700 border-b">Name</th>
+          <th class="px-6 py-3 bg-gray-200 text-gray-700 border-b">Measurement Unit</th>
+          <th class="px-6 py-3 bg-gray-200 text-gray-700 border-b">Unit Price</th>
+          <th class="px-6 py-3 bg-gray-200 text-gray-700 border-b">Actions</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @foods.each do |food| %>
+          <tr>
+            <td class="ml-2 px-6 py-4 border border-gray-200"><%= food.name %></td>
+            <td class="ml-2 px-6 py-4 border border-gray-200"><%= food.measurement_unit %></td>
+            <td class="ml-2 px-6 py-4 border border-gray-200">$<%= food.price %></td>
+            <td class="ml-2 px-6 py-4 border border-gray-200">
+              <%= link_to 'Delete', food_path(food), method: :delete, data: { confirm: 'Are you sure?' }, class: "text-red-600 ml-2" %>
```

Comment:
Great job😍😍 Adding the `tailwind` CSS style, However, when I tried to delete each food it doesn't delete instead goes to the food details path. but expected behavior should delete food on the same page means the food#index page